### PR TITLE
Finalizing module

### PR DESCRIPTION
**Pull request by alibazlamit**

### CODE REVIEW

**Comment by nurfet-becirevic** [comment]
Error when importing the module:

```
PS C:\Users\Nur> Import-Module OneAndOne
Import-Module : The module to process 'Profitbricks.dll', listed in field
'ModuleToProcess/RootModule' of module manifest
'C:\Users\Nur\Documents\WindowsPowerShell\Modules\OneAndOne\OneAndOne.psd1'
was not processed because no valid module was found in any module
directory.
At line:1 char:1
+ Import-Module OneAndOne
+ ~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ResourceUnavailable: (OneAndOne:String)
[Import-Module], PSInvalidOperationException
    + FullyQualifiedErrorId :
Modules_ModuleFileNotFound,Microsoft.PowerShell.Commands.ImportModuleCommand
```

**Comment by alibazlamit** [comment]
@nurfet-becirevic  Please pull latest from master and try again, i have fixed the issue.

File: OneAndOne/Oneandone.cs [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+        #endregion
+
+        protected override void BeginProcessing()
+        {
+            try
+            {
+                var config = new Configuration
+                {
+                    //ApiKey = "c4a21f145229f0597d60b0e531cfc69f",
```
Comment (nurfet-becirevic) [inline OneAndOne/Oneandone.cs]: Remove the API key

File: OneAndOne/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("OneAndOne")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("OneAndOne")]
+[assembly: AssemblyCopyright("Copyright ©  2017")]
```
Comment (nurfet-becirevic) [inline OneAndOne/Properties/AssemblyInfo.cs]: Copyright (c) 2017 1&1 Internet SE

File: OneAndOne/Role.cs [hunk 3]
```diff
@@ -0,0 +1,610 @@
...
+                {
+                    var users = roleApi.GetRoleUsers(RoleId);
+                    WriteObject(users);
+                }
+                else
+                {
+                    var userInfo = roleApi.ShowRoleUser(RoleId, UserId);
+                    WriteObject(userInfo);
+                }
+
```
Comment (nurfet-becirevic) [inline OneAndOne/Role.cs]: too many empty lines

File: README.md [hunk 4]
```diff
@@ -1,2 +1,1049 @@
...
+
+To create a user and generate an API token that will be used to authenticate against the REST API, log into your 1&1 control panel. Go to the Server section -> Management -> Users.
+
+## Installation
+
+OneAndOne Powershell Module requires .NET Framework 4.5 or higher.  You can download .NET Framework version 4.5 from [here](https://www.microsoft.com/en-us/download/details.aspx?id=40779).
+
+Download the [OneAndOne.zip](https://github.com/1and1/oneandone-powershell/releases/download/v1.0.0/OneAndOne.zip) file and extract all. Use one of the following options to make the module available inside PowerShell:
+
+1. Place the resulting folder `OneAndOne` in `%USERPROFILE%\Documents\WindowsPowerShell\Modules\` to auto-load the module on PowerShell start for a specific user.
```
Comment (nurfet-becirevic) [inline README.md]: The module do not auto-load. `Import-Module OneAndOne` is required. Please add it.

File: OneAndOne/Common.cs [hunk 5]
```diff
@@ -0,0 +1,438 @@
+﻿using OneAndOne.Client;
+using OneAndOne.POCO;
+using OneAndOne.POCO.Response.Common;
+using OneAndOne.POCO.Response.DataCenters;
+using OneAndOne.POCO.Response.DVDS;
+using OneAndOne.POCO.Response.ServerAppliances;
+using OneAndOne.POCO.Response.Usages;
```
Comment (alibazlamit) [inline OneAndOne/Common.cs]: These API operations are grouped to not have too many classes, each API operation has one or two endpoint, it does not make any difference when you use the commands from powershell.

**Pull request merged**